sklearn/linear_model/logistic.py

```diff
@@ -233,7 +227,8 @@ def _multinomial_loss(w, X, Y, alpha, sample_weight):
 
     Parameters
     ----------
-    w : ndarray, shape (n_classes * n_features,) or (n_classes * (n_features + 1),)
+    w : ndarray,
+        shape (n_classes * n_features,) or (n_classes * (n_features + 1),)
```

Comment:
It would render ok with a ``\`` I think. But I really don't care that much. I would have been fine with leaving it as it was. Actually now it should at least be somewhat indented, I think?